### ISSUE TITLE
Loot Sheet not functioning for players, no ability to set permissions, visible to GM only

### ISSUE THREAD

**Issue by grepheavy** [jopeek/fvtt-loot-sheet-npc-5e#387]
**Describe the bug**
Players are unable to access any loot sheets, double clicking the actor does nothing and doesnt load anything.  In GM mode no permissions are accessible or able to be set.  Can change from loot to merchant and load items but they arent even accessible to any player

**To Reproduce**
1. Create a simple loot sheet/actor
2. load into map/scene
3. load PC next to
4. as PC double click on lootsheet actor
5. no results

**Expected behavior**
basically function as normal

**Screenshots**
If applicable, add screenshots to help explain your problem

**Desktop (please complete the following information):**
 - OS: windows
 - Browser: chrome, safari, edge

![lootsheeterrors](https://user-images.githubusercontent.com/92830437/217333212-6f18704d-6231-4c6b-9467-3a5ca156a50d.JPG)

**Comment by jopeek** [comment]
You have to set permissions same as any other character. It’s not part of the sheet anymore. Observer let’s them open and purchase/loot.

**Comment by grepheavy** [comment]
Thanks,  so divided looting doesnt work?

And one has to change the loot permissions live during the game because if they do it ahead of time players can see the loot before even playing and can loot from the actors menu.

**Comment by jopeek** [comment]
Divided loot may work? I haven’t tested that. It’s just based on the permissions. As for having to do it live, yes. Not ideal, I know, but that’s the current state.

**Comment by arcanjl** [comment]
how do we change the permissions? The only way I know how is the in the actor's section, and that is by changing the prototype, which I don't want to do. Is there a way to change the token by itself?

**Comment by jopeek** [comment]
I don’t think so, no.

**Comment by sgiath** [comment]
With this change, this module is pretty much useless - either players cannot open the loot sheet or they can see whole stats for the creature in the Actors tab 🤷🏻

**Comment by jopeek** [comment]
What actors tab? There’s no actors tab on the lootsheet.

**Comment by Ferliczkoni** [comment]
@jopeek I have given my players the observer rights, and they now can open the chest but the loot button is not working.

**Comment by jopeek** [comment]
Can they see it and click it but nothing happens? If so, I will have to review the loot button code. I haven’t checked it in a while.

**Comment by Ferliczkoni** [comment]
Yes, that's the case. They can see it but pressing the loot button does nothing at all.

**Comment by jopeek** [comment]
Ok I’ll try to fix that this week sometime.

**Comment by sgiath** [comment]
@jopeek I used it as follows:
1. have a creature template in Actors (with items and money attached), create a bunch of unlinked tokens on the map, and run combat with them
2. once the creature is defeated, change the sheet to Loot sheet and give users permission to loot it

Now I have to give them permission to access the actor template so they can loot tokens on the map, but if they go to the "Actors" tab on the right they can see creatures' whole stats.

**Comment by jopeek** [comment]
I see what you’re saying. That was always a problem, wasn’t it? The on-sheet permission controls did the exact same thing. I don’t think it’s possible to give permissions only to a specific token and not the actor…

**Comment by sgiath** [comment]
I would swear it was working before but my memory is not perfect :)

**Comment by grepheavy** [comment]
> What actors tab? There’s no actors tab on the lootsheet.
> […](#)
> On Mar 6, 2023 at 5:47 AM -0800, Filip Vavera ***@***.***>, wrote: With this change, this module is pretty much useless - either players cannot open the loot sheet or they can see whole stats for the creature in the Actors tab 🤷🏻 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***>

if i give them permission to see the actor for the loot, then they can easily see that actor in their "actors" panel, its not hidden in any way.

**Comment by tingle-berry** [comment]
Still getting this issue, has there been a fix yet?

vtt version: 10.291
loot sheet version: 5.06

![image](https://github.com/jopeek/fvtt-loot-sheet-npc-5e/assets/132842836/7790f2b9-0ce0-4088-9674-177f6b4bdcbe)

**Comment by jopeek** [comment]
No. I don’t consider this a bug but a missing feature which I have no time to add.

**Comment by jopeek** [comment]
Permissions are back
